pkg/config/config.go

```diff
@@ -405,8 +405,9 @@ func newConfigGroup() *cb.ConfigGroup {
 	}
 }
 
-// newEnvelope creates an unsigned envelope of type txType using with the marshalled
-// cb.ConfigGroupEnvelope proto message.
+// newEnvelope creates an unsigned envelope of the desired type containing
+// a payload Header and the marshaled proto message as the payload Data.
+// message
```

Comment:
Did you forget to delete this line?

Reply:
Ah yep, didn't even notice it for a while after reading your comment. :) Removed.